bern-university-of-applied-sciences-institute-for-digital-economy-in-the-construction-and-wood-industry-idbh.csl

```diff
@@ -0,0 +1,341 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <!-- version "BFH-AHB_11-23_Deutsch; Elsevier - Harvard (author-date)", wht2 -->
+    <title>BFH-AHB_Deutsch</title>
+    <title-short>BFH-AHB (De)</title-short>
```

Comment:
Spell out the title in full following the requirements here: https://github.com/citation-style-language/styles/blob/master/STYLE_REQUIREMENTS.md

Also, the title says "Deutsch" but the default locale is "en-US" and the various hardcoded words in the style are in English. Please correct whichever of those is incorrect.